Resources/Prototypes/Nyanotrasen/Entities/Mobs/NPCs/fungi.yml

```diff
@@ -0,0 +1,82 @@
+- type: entity
+  parent: SimpleSpaceMobBase
+  id: MobFungusBase
+  name: fungus
+  abstract: true
+  components:
+  - type: Bloodstream
+    bloodReagent: Mold
+  - type: PsionicInsulation
+
+- type: entity
+  parent: MobFungusBase
+  id: MobFungusGasSpore
+  name: gas spore
+  description: A floating ball of gas.
+  components:
+  - type: CombatMode
+    combatToggleAction:
+      enabled: false
+      autoPopulate: false
+      name: action-name-combat
+  - type: GasCanister
+    gasMixture:
+      volume: 200
+      moles:
+        - 0 # oxygen
+        - 0 # nitrogen
+        - 0 # CO2
+        - 0 # Plasma
+        - 0 # Tritium
+        - 0 #  Water vapor
+        - 300 # Miasma
+      temperature: 293.15
+  - type: HTN
+    rootTask: SelfExploderCompound
+  - type: Faction
+    factions:
+      - SimpleHostile #TODO: Fungi faction?
```

Comment:
A fungi faction on the base should be okay. This way you can have them attacking other NPCs. Maintaining all the factions that are hostile to one another is a small chore though.

Reply:
it doesn't need to be in this PR though, especially when we're unsure of how exactly it should be implemented